templates/serve-stable-diffusion/README.md

```diff
@@ -0,0 +1,52 @@
+# Serving a Stable Diffusion Model with Ray Serve
+This template shows you how to develop and test the model locally and deploy it into production.
+
+## Step 1: Install python dependencies
+```
+pip install diffusers==0.25.0 transformers==4.36.2 accelerate==0.25.0
+```
+
+## Step 2: Deploy the model locally
+- Open a new terminal (ctl+shift+`) in VS Code. 
+- Run the command below to deploy your model at http://localhost:8000.  
+
+```bash
+serve run app:entrypoint
+```
+
+## Step 3: Send test requests to the running model
+- Open a new terminal. Run the command below to send a request to your model. 
+- An image should be generated in the current directory
+```bash
+python query.py
+```
+
+--------
+##  Note: the following steps are still under implementation. We'll send a notice after they are ready for test.
+--------
+
+## Step 4: Deploy the model as an Anyscale Service
+Deploy it as an Anyscale Service for staging or production traffic with `--publish` flag
+
+```bash
+serve run app:entrypoint --publish
```

Comment:
Maybe also add the --name argument here?

Reply:
Hmm. Why do we need the --name argument? 
IIRC, you don't need it if it's within the workspace. You only need it If it's outside workspace?